package.json

```diff
@@ -61,7 +62,11 @@
     "body-parser": "^1.18.3",
     "express": "^4.16.4",
     "faker": "^4.1.0",
+    "fast-csv": "^2.4.1",
```

Comment:
Great- streams will definitely help with processing large amounts of data. I'm curious to see how this library compares to a single custom written function in terms of time complexity.